app/services/submission/SubmissionService.scala

```diff
@@ -154,7 +148,7 @@ class SubmissionService @Inject() (
         case _                                            => placeOfLoading
       },
       HouseConsignment = houseConsignments match {
-        case Nil => Seq(HouseConsignmentType06("1", Nil))
+        case Nil => Seq(HouseConsignmentType06(BigInt("1"), Nil))
```

Comment:
Think `BigInt("1")` can just be `1`. The compiler should be able to handle interpreting an Int as a BigInt I believe.